components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -82,26 +123,27 @@ function FeedbackComponent(props: { onClose: () => void; onSubmit: () => void; i
                                 .
                             </p>
                         </div>
-                    </div>
```

Comment:
Restoring text alignment from https://github.com/gitpod-io/gitpod/pull/10071#discussion_r880288290.

Suggestion for https://github.com/gitpod-io/gitpod/pull/10071/files#diff-51811862e7996f416f9f7d73ff24341eecd96f3c083fb46d1b00b6cf94b50686R117:

```js
text-gray-500 text-left
```